app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -221,7 +226,9 @@ export function createElectronBuilderConfig(passedArgs: Arguments): electronBuil
                 console.log('  • Notarizing.')
                 await electronNotarize.notarize({
                     // This will always be defined since we set it at the top of this object.
-                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    // The type-cast is safe because this is only executes
+                    // when `platform === electronBuilder.Platform.MAC`.
+                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, no-restricted-syntax
                     appBundleId: (buildOptions as macOptions.MacConfiguration).appId!,
```

Comment:
type assertion #2 (i'll be pointing them out to make reviewing more convenient)

Reply:
Thanks so much for making the review so easy!